chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/RoomSpek.kt

```diff
@@ -121,4 +124,61 @@ class RoomEventsSpek : Spek({
 
     }
 
+    describe("currentUser '$PUSHERINO'") {
+
+        it("creates room") {
+            setUpInstanceWith(newUsers(PUSHERINO, ALICE))
+
+            val pusherino = chatFor(PUSHERINO).connect().wait().assumeSuccess()
+
+            val room = pusherino.createRoom(GENERAL).wait().assumeSuccess()
+
+            assertThat(room.name).isEqualTo(GENERAL)
+        }
+
+        xit("updates room name", reason = "Requires elevated access") {
```

Comment:
What's this one blocked on (assuming that's what the `xit` means)?

Reply:
This requires admin access. I didn't manage to get it to have admin rights :/

Reply:
There's an `admin` role automatically created for every chatkit instance, so you could assign that to one of your users and use that to delete / update rooms